ui/src/app/modules/settings/settings.component.html

```diff
@@ -45,6 +45,7 @@ <h5 class="primary-text mt-3" translate="settings.network.title_network"></h5>
         <select class="custom-select" [formControl]="legacyMdnsFormControl"
           [attr.aria-label]=" 'settings.mdns_advertiser' | translate">
           <option value="bonjour-hap">Bonjour HAP</option>
+          <option value="avahi">Avahi</option>
```

Comment:
We could only display this option if the user is running Linux.

Reply:
Maybe having a way for the UI to check if Avahi is available would be useful, as when running Docker the "avahi" service is only started on the Ubuntu variant of the image by default.

Reply:
Sounds good!

Reply:
Can we interface with homebridge to report which advertiser was picked in "choose default" mode? see https://github.com/homebridge/HAP-NodeJS/pull/922